src/__tests__/lib/item-input/array.test.ts

```diff
@@ -598,34 +602,37 @@ describe('checkboxDef', () => {
 
 		it('uses supplied validate method', async () => {
 			const validate = jest.fn<Required<CheckboxDefOptions<string>>['validate']>()
-			const def = checkboxDef<string>('Checkbox Def', ['Item 1', 'Item 2', 'Item 3'], { validate })
+			const def = checkboxDef('Checkbox Def', ['Item 1', 'Item 2', 'Item 3'], { validate })
 			promptMock.mockResolvedValueOnce({ values: simpleResults })
 
 			expect(await def.buildFromUserInput()).toBe(simpleResults)
 
-			expect(promptMock).toHaveBeenCalledTimes(1)
-			expect(promptMock).toHaveBeenCalledWith(expect.objectContaining({
+			expect(promptMock).toHaveBeenCalledExactlyOnceWith(expect.objectContaining({
 				validate,
 			}))
 		})
 
 		it('includes help option when helpText is supplied', async () => {
-			const def = checkboxDef<string>('Checkbox Def', ['Item 1', 'Item 2'], { helpText: 'help text' })
+			const def = checkboxDef<string>(
+				'Checkbox Def',
+				['Item 1', 'Item 2'],
+				{ helpText: 'help text' },
+			)
 			promptMock.mockResolvedValueOnce({ values: ['Item 1'] })
 
 			expect(await def.buildFromUserInput()).toStrictEqual(['Item 1'])
 
-			expect(promptMock).toHaveBeenCalledTimes(1)
-			expect(promptMock).toHaveBeenCalledWith(expect.objectContaining({
+			expect(promptMock).toHaveBeenCalledExactlyOnceWith(expect.objectContaining({
 				type: 'checkbox',
 				name: 'values',
 				message: 'Select Checkbox Def.',
 				choices: ['Item 1', 'Item 2'],
 				default: finishAction,
+			}))
+			expect(promptMock).toHaveBeenCalledWith(expect.not.objectContaining({
 				validate: undefined,
 			}))
 
-			expect(consoleLogSpy).toHaveBeenCalledTimes(1)
 			expect(consoleLogSpy).toHaveBeenCalledWith('\nhelp text\n')
```

Comment:
Same as above: should this be `toHaveBeenCalledExactlyOnceWith`?

Reply:
Same answer as above.